INPUT_SCHEMA.json

```diff
@@ -65,7 +66,7 @@
       "editor": "hidden"
     },
     "checkInDate": {
-      "title": "Check in date",
+      "title": "Check in date (YYYY-MM-DD)",
```

Comment:
add the `pattern` here `\\d{4}-\\d{2}-\\d{2}`

Reply:
Done